xblock/core.py

```diff
@@ -105,6 +105,20 @@ def open_local_resource(cls, uri):
 
         return pkg_resources.resource_stream(cls.__module__, os.path.join(cls.resources_dir, uri))
 
+    @property
+    def supports_save(self):
```

Comment:
I considered the no-op ``save`` but that seems ugly to me. I intended to make this a class attribute and didn't think through what ``@property`` meant, so I'll switch it and see how that seems.